src/Assent/Assent.csproj

```diff
@@ -30,4 +30,8 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />  </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Just because I'm pedantic and like to keep it consistent with my other projects, could you please move this group above the filtered ItemGroups.